.github/workflows/auto-label-pr.yml

```
@@ -0,0 +1,65 @@
+  pull_request:
+    types: [opened]
+    branches:
+      - develop
+
+jobs:
+  add-wip-label:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+      issues: write
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)